packages/headless/src/ssr-commerce.index.ts

```diff
@@ -362,6 +362,9 @@ export {
   buildQueryRankingExpressionSortCriterion,
   buildRelevanceSortCriterion,
 } from './features/sort-criteria/criteria.js';
+export type {SortCriterion as SortSortCriterion} from './features/sort/sort.js';
+
+export {SortBy as SortSortBy} from './features/sort/sort.js';
```

Comment:
I'm not sure what could be better names to distinguish these from the sort-criteria.js exports

Reply:
I was ending up with a hydration error, but I just figured out why